scalar_expressions/func_expr.py

```diff
@@ -1,20 +1,20 @@
-from scalar_expression_base import scalar_expression_base
+from .scalar_expression_base import scalar_expression_base
 from variable_types.scoreboard_var import scoreboard_var
 from environment import isInt
 import math
 
 def factor(n):
-	i = 2
-	limit = math.sqrt(n)    
-	while i <= limit:
-	  if n % i == 0:
-		yield i
-		n = n / i
-		limit = math.sqrt(n)   
-	  else:
-		i += 1
-	if n > 1:
-		yield n
+    i = 2
+    limit = math.sqrt(n)
+    while i <= limit:
+        if n % i == 0:
+            yield i
+            n = n // i  # Use integer division for Python 3
```

Comment:
If we're adding comments like this, either be consitent and add them everywhere we're doing this change, or nowhere.

Reply:
nowhere please, this is annoying and useless

Reply:
Honestly, this function is only used in a deprecated unit test, and should be removed, anyway.